### PR TITLE
Implement vale in pyfluent-visualization

### PR DESCRIPTION
**Pull request by RobPasMue**
Hi @PipKat! 

Following https://github.com/pyansys/pyfluent/pull/655, I have open this new PR to implement vale in the pyfluent-visualization repo. Let me know if you have any problems. Just, check out my branch and start implementing there the vale checks.

Thanks in advance! 

_As it is being done in other PyAnsys projects, vale has been implemented. Vale is a linter which eases the review process to the documentation team by detecting common mistakes and forcing the developer to follow the Google style guidelines when writing the docs._

### CODE REVIEW

**Comment by PipKat** [comment]
@dnwillia-work All checks have passed. As I indicated in chat, the edits made in this PR are only the ones needed to get Vale implemented. Would you please assign another reviewer so that we can get this PR merged? I see you have an issue opened here about all the doc changes that need to be made. Once I complete a PR adding Vale info to the Python Developer's Guide, I can add a link to your issue. The content I'm adding explains how to run Vale locally and fix the issues that are found so that your PR can be completed. @RobPasMue Just tagging you so that you see that I've resolved the Vale issues.

**Comment by dnwillia-work** [comment]
> @dnwillia-work All checks have passed. As I indicated in chat, the edits made in this PR are only the ones needed to get Vale implemented. Would you please assign another reviewer so that we can get this PR merged? I see you have an issue opened here about all the doc changes that need to be made. Once I complete a PR adding Vale info to the Python Developer's Guide, I can add a link to your issue. The content I'm adding explains how to run Vale locally and fix the issues that are found so that your PR can be completed. @RobPasMue Just tagging you so that you see that I've resolved the Vale issues.

Done, I've added Aseem and Sean, either of them can approve.

**Review by dnwillia-work — APPROVED** [review]

File: doc/source/api/visualization/index.rst [hunk 1]
```diff
@@ -3,13 +3,13 @@
 Visualization
 =============
 
-Post-processing of Fluent results can be done with either Fluent in-built
-post-processing capabilities or with the PyVista/MatplotLib integration.
+Postprocessing of Fluent results can be done with either built-in Fluent
+postprocessing capabilities or with a PyVista or Matplotlib integration.
```
Comment (dnwillia-work) [inline doc/source/api/visualization/index.rst:7]:
```suggestion
postprocessing capabilities, PyVista, or Matplotlib integration.
```
?

Comment (PipKat) [reply doc/source/api/visualization/index.rst:7]: @dnwillia-work Isn't the "integration" only for PyVista or Matplotlib given that the other option is built-in Fluent postprocessing capabilities. Perhaps I'm misunderstanding what's intended.

**Comment by RobPasMue** [comment]
> @dnwillia-work All checks have passed. As I indicated in chat, the edits made in this PR are only the ones needed to get Vale implemented. Would you please assign another reviewer so that we can get this PR merged? I see you have an issue opened here about all the doc changes that need to be made. Once I complete a PR adding Vale info to the Python Developer's Guide, I can add a link to your issue. The content I'm adding explains how to run Vale locally and fix the issues that are found so that your PR can be completed. @RobPasMue Just tagging you so that you see that I've resolved the Vale issues.

Thanks @PipKat, LGTM!

**Comment by PipKat** [comment]
@ajain-work and @seanpearsonuk  We are looking for another person to review and approve this PR. Thanks!

**Review by ajain-work — APPROVED** [review]

File: doc/source/api/visualization/index.rst (hunk 1 above)
Comment (RobPasMue) [reply doc/source/api/visualization/index.rst:7]: If I understood correctly, you can use either of the three backends for post-processing results. Commiting suggestion and merging :)

**Pull request merged**